extensions/hibernate-orm/deployment/pom.xml

```diff
@@ -58,6 +58,11 @@
             <artifactId>quarkus-hibernate-orm-derby-deployment</artifactId>
             <optional>true</optional>  <!-- conditional dependency -->
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-hibernate-validator-spi</artifactId>
+            <optional>true</optional>  <!-- conditional dependency -->
```

Comment:
Usually SPI modules aren't optional, because the presence of the SPI doesn't mean that the actual extension that leverages the build items (in this case `quarkus-hibernate-validator-deployment`)

Reply:
It should yeah :)